### PR TITLE
Fix all accessibility errors

### LINKED ISSUE #327: a11y issues
we need to fix a11y issues -- on the home page, there are 38 contrast issues and one svg missing alt text.  there are issues with other pages as well.  I will add a checklist here in the near future

### PR DESCRIPTION
**Pull request by smorimoto**
Closes #327

### CODE REVIEW

File: _layouts/menu.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
       <svg
         xmlns="http://www.w3.org/2000/svg"
         viewBox="0 0 100 100"
-        role="img"
+        role="button"
```
Comment (smorimoto) [inline _layouts/menu.html:118]: `img` was not appropriate.

File: _layouts/page_footer.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h3 id="connect-label">{{ site["footer"]["connect"] }}</h3>
                   viewBox="0 0 16 16"
                   version="1.1"
                   width="24"
-                  role="img"
+                  role="link"
```
Comment (smorimoto) [inline _layouts/page_footer.html:17]: `img` was not appropriate.

File: _layouts/page_footer.html [hunk 3]
```diff
@@ -32,7 +32,7 @@ <h3 id="connect-label">{{ site["footer"]["connect"] }}</h3>
...
```
Comment (smorimoto) [inline _layouts/page_footer.html:35]: `img` was not appropriate.

File: _layouts/page_footer.html [hunk 4]
```diff
@@ -49,7 +49,7 @@ <h3 id="connect-label">{{ site["footer"]["connect"] }}</h3>
...
```
Comment (smorimoto) [inline _layouts/page_footer.html:52]: `img` was not appropriate.

File: _layouts/page_footer.html [hunk 5]
```diff
@@ -72,7 +72,7 @@ <h3 id="connect-label">{{ site["footer"]["connect"] }}</h3>
...
```
Comment (smorimoto) [inline _layouts/page_footer.html:75]: `img` was not appropriate.

File: _layouts/page_footer.html [hunk 6]
```diff
@@ -92,7 +92,7 @@ <h3 id="connect-label">{{ site["footer"]["connect"] }}</h3>
...
```
Comment (smorimoto) [inline _layouts/page_footer.html:95]: `img` was not appropriate.

File: _sass/_featurelist.scss [hunk 7]
```diff
@@ -151,7 +151,6 @@
       margin: 0;
       clear: both;
       font-weight: 800;
-      color: grey;
```
Comment (smorimoto) [inline _sass/_featurelist.scss:154]: Must meet WCAG 2 AA contrast ratio thresholds

File: _sass/_variables.scss [hunk 8]
```diff
@@ -1,6 +1,6 @@
 // Colors
 $color-tc39: #f90;
-$color-tc39-dark: #ab6700;
```
Comment (smorimoto) [inline _sass/_variables.scss:3]: Must meet WCAG 2 AA contrast ratio thresholds

File: _sass/_variables.scss [hunk 9]
```diff
@@ -25,6 +25,6 @@ $bp-desktop: 1200px;
 
 // Tags
 $tag-tests: #017e78;
-$tag-spec: #ab6700;
```
Comment (smorimoto) [inline _sass/_variables.scss:28]: Must meet WCAG 2 AA contrast ratio thresholds

**Review by ctcpip — APPROVED** [review]

**Pull request merged**